docs/contributors/testing-overview.md

```diff
@@ -368,6 +368,10 @@ If you're using a different setup, you can provide the base URL, username and pa
 WP_BASE_URL=http://localhost:8888 WP_USERNAME=admin WP_PASSWORD=password npm run test-e2e
 ```
 
+### New Core Blocks
+
+Every core block is required to have at least one set of fixture files to test the parsing and serialization of that block. See [the e2e tests fixtures readme](/packages/e2e-tests/fixtures/blocks/README.md) for more information and instructions.
```

Comment:
I think this is a bit out of date and we now also want fixtures for each deprecation (@gziolo added lots of those fixtures for existing blocks a while back). The linked file also has a similar sentence that doesn't mention that. Maybe we can briefly update both sentences to something like this:
```suggestion
Every core block is required to have at least one set of fixture files for its main save function and one for each deprecation. These fixtures test the parsing and serialization of the block. See [the e2e tests fixtures readme](/packages/e2e-tests/fixtures/blocks/README.md) for more information and instructions.
```

Reply:
This is a section for new core block so this might be a valid statement. 

Moving forward, we also want to have fixtures for all deprecations introduced to ensure we keep backward compatibility forever.

Maybe we extend the following sentence?
```diff
- and one for each deprecation.
+ and one for each deprecation added later during its lifetime.
```

Reply:
I think there is something with transforms now as well, which is based on the fixtures. @jorgefilipecosta should know more. I think we need to revisit that though, as we introduce too many checks for blocks which support multiple options to transform.